eng/liveBuilds.targets

```diff
@@ -185,9 +185,12 @@
         $(LibrariesNativeArtifactsPath)*.dwarf;
         $(LibrariesNativeArtifactsPath)*.pdb"
         IsNative="true" />
-      <LibrariesRuntimeFiles Condition="'$(TargetsMobile)' == 'true'"
+      <LibrariesRuntimeFiles Condition="'$(TargetOS)' == 'Browser'"
                              Include="
-        $(LibrariesNativeArtifactsPath)**\*.*" 
+        $(LibrariesNativeArtifactsPath)wasm\runtimes\debug\dotnet.js;
```

Comment:
What is the problem with wildcards here ? These files contain files created by the build, its not going to pick up random files. It also means every time some files are added to the build, this needs to be changed as well.

Reply:
It's very easy to accidentally add some temp file or other build artifact that we don't intend to ship.

I can change it to *.js and *.wasm if you think we'll have more files here.

Reply:
I also prefer explicit names, it's much easier to grep for them to see how they are packaged